### ISSUE TITLE
Remaning slash extension use in book example

### ISSUE THREAD

**Issue by martinpub** [w3c/a11y-discov-vocab#73]
Hi! When reading through this document, I saw that the example towards the end has the accessibilityFeature value "displayTransformability/resizeText", see https://github.com/w3c/a11y-discov-vocab/blob/main/index.html#L1720. Judging from #53, the slash extension mechanism is no longer recommended, so perhaps the example should also be updated?

**Comment by mattgarrish** [comment]
Yes, clearly looks like we missed that one use of a slash. Thanks for the report!
